### PR TITLE
Feat (#7561) Announcements are not supported for MLModel from UI

### LINKED ISSUE #7561: Announcements are not supported for MLModel from UI
**Affected module**
Does it impact the UI, backend or Ingestion Framework?

**Describe the bug**

**To Reproduce**

Screenshots or steps to reproduce

**Expected behavior**

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
Closes #7561 
### Describe your changes :

I worked on adding support for the following things for the MLModel from UI.
- Conversation
- Tasks
- Announcements

#
### Type of change :

- [x] New feature

#
### Frontend Preview (Screenshots) :

https://user-images.githubusercontent.com/59080942/196106001-8e6389e8-c001-4233-9b8d-ab65be7d18e9.mov

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/EntityAnnouncement.spec.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { getCurrentLocaleDate, getFutureLocaleDateFromCurrentDate } from "../../../src/utils/TimeUtils";
+import { descriptionBox, login, visitEntityDetailsPage } from "../../common/common";
+import { LOGIN, SEARCH_ENTITY_DASHBOARD, SEARCH_ENTITY_PIPELINE, SEARCH_ENTITY_TABLE, SEARCH_ENTITY_TOPIC } from "../../constants/constants";
+
+const ANNOUNCEMENT_ENTITIES=[SEARCH_ENTITY_TABLE.table_1,SEARCH_ENTITY_TOPIC.topic_1,SEARCH_ENTITY_DASHBOARD.dashboard_1,SEARCH_ENTITY_PIPELINE.pipeline_1]
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/EntityAnnouncement.spec.js:18]: move this to const

File: openmetadata-ui/src/main/resources/ui/src/components/MlModelDetail/MlModelDetail.component.test.tsx [hunk 2]
```diff
@@ -200,6 +221,11 @@ jest.mock('../common/CustomPropertyTable/CustomPropertyTable', () => ({
     .mockReturnValue(<p>CustomPropertyTable.component</p>),
 }));
 
+window.IntersectionObserver = jest.fn().mockImplementation(() => ({
+  observe: jest.fn(),
+  unobserve: jest.fn(),
+}));
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/MlModelDetail/MlModelDetail.component.test.tsx:227]: Let's move this to `https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-ui/src/main/resources/ui/src/setupTests.js`

File: openmetadata-ui/src/main/resources/ui/src/components/MlModelDetail/MlModelDetail.component.tsx [hunk 3]
```diff
@@ -509,12 +562,30 @@ const MlModelDetail: FC<MlModelDetailProp> = ({
                 </Fragment>
               )}
               {activeTab === 2 && (
+                <div
+                  className="tw-py-4 tw-px-7 tw-grid tw-grid-cols-3 entity-feed-list tw--mx-7 tw--my-4"
+                  id="activityfeed">
+                  <div />
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/MlModelDetail/MlModelDetail.component.tsx:568]:
```suggestion

```

File: openmetadata-ui/src/main/resources/ui/src/components/MlModelDetail/MlModelDetail.component.tsx [hunk 4]
```diff
@@ -509,12 +562,30 @@ const MlModelDetail: FC<MlModelDetailProp> = ({
...
+                    isEntityFeed
+                    withSidePanel
+                    deletePostHandler={deletePostHandler}
+                    entityName={mlModelDetail.name}
+                    feedList={entityThread}
+                    postFeedHandler={postFeedHandler}
+                    updateThreadHandler={updateThreadHandler}
+                    onFeedFiltersUpdate={handleFeedFilterChange}
+                  />
+                  <div />
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/MlModelDetail/MlModelDetail.component.tsx:579]:
```suggestion

```

File: openmetadata-ui/src/main/resources/ui/src/pages/MlModelPage/MlModelPage.component.tsx [hunk 5]
```diff
@@ -202,10 +238,57 @@ const MlModelPage = () => {
     }
   };
 
+  const getEntityFeedCount = () => {
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/MlModelPage/MlModelPage.component.tsx:241]: we are not returning anything so name should be `fetchEntityFeedCount` instead `getEntityFeedCount`

File: openmetadata-ui/src/main/resources/ui/src/pages/MlModelPage/MlModelPage.component.tsx [hunk 6]
```diff
@@ -202,10 +238,57 @@ const MlModelPage = () => {
...
+    getFeedCounts(
+      EntityType.MLMODEL,
+      mlModelFqn,
+      setEntityFieldThreadCount,
+      setEntityFieldTaskCount,
+      setFeedCount
+    );
+  };
+
+  const getFeedData = async (
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/MlModelPage/MlModelPage.component.tsx:251]:
```suggestion
  const fetchFeedData = async (
```

File: openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/TaskDetailPage/TaskDetailPage.tsx [hunk 7]
```diff
@@ -506,252 +510,263 @@ const TaskDetailPage = () => {
...
-                      updateThreadHandler={onTaskFeedUpdate}
+                  </div>
+                </Card>
+
+                <Card
+                  data-testid="task-data"
+                  style={{
+                    ...cardStyles,
+                    marginTop: '16px',
+                    marginLeft: '24px',
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/TaskDetailPage/TaskDetailPage.tsx:640]: Can we you classes instead inline styles?

**Comment by Sachin-chaurasiya** [comment]
Thanks, @chirag-madlani, All comments are addressed.

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**